indico/modules/events/management/templates/event_permissions_dialog.html

```diff
@@ -0,0 +1,61 @@
+<form id="js-permissions-dialog-form">
+    <input type="hidden" id="selected_permissions" value='{{ permissions|tojson|forceescape }}' name="selected_permissions">
+    <input type="hidden" id="principal" value='{{ principal|tojson|forceescape }}'>
+    <div class="permissions-dialog">
+        <ul class="permissions">
+            {% for code, tree_elem in permissions_tree.viewitems() recursive %}
+                <li class="item">
+                    <input type="checkbox" id="cb-{{ code }}"
+                           class="circle-checkbox" {{ 'checked' if code in permissions or code == '_read_access' }}
+                           {{ 'disabled' if code == '_read_access' }}
+                           data-code="{{ code }}">
+                    <label for="cb-{{ code }}" title="{{ tree_elem.description }}">{{ tree_elem.title }}</label>
+                    {% if tree_elem.children %}
+                        <ul class="permissions">
+                            {{ loop(tree_elem.children|dictsort(by='key')) }}
+                        </ul>
+                    {% endif %}
+                </li>
+            {% endfor %}
+        </ul>
+        <div class="toolbar f-j-end">
+            <button type="submit" class="i-button big highlight" data-disabled-until-change>{% trans %}Save{% endtrans %}</button>
+            <button class="i-button big" data-button-back>{% trans %}Cancel{% endtrans %}</button>
+        </div>
+    </div>
+</form>
+<script>
+    (function () {
+        'use strict';
+
+        function updateHiddenField() {
+            var data = [];
+            var $form = $('#js-permissions-dialog-form');
+            $form.find('[type=checkbox]:checked').each(function() {
```

Comment:
*Way* too generic. Should be restricted to some container related to the permission dialog.

Also, use `input:checkbox` or `input[type=checkbox]` to select checkboxes.

Reply:
Isn't it already contained in the dialog?
```
var $form = $('#js-permissions-dialog-form');
```

Reply:
Yes, indeed. But I think somewhere below there's one where you have `$('[type=checkbox]')`